st3/sublime_lib/pure_resource_path.py

```diff
@@ -0,0 +1,103 @@
+import posixpath
+from functools import total_ordering
+
+
+__all__ = ['PureResourcePath']
+
+
+@total_ordering
+class PureResourcePath():
+    """A representation of a Sublime Text resource path. Inspired by pathlib."""
```

Comment:
Should mention that only `/` are supported, not `\`.

And that not all methods of pathlib.PurePath are implemented, but those that make sense.

Reply:
Added info.